test/unit/geojson.js

```diff
@@ -16,12 +16,12 @@ function parse(geojson) {
 describe('GeoJsonParser', function () {
```

Comment:
Could you add some tests with maybe more features, but testing with all the new options or not in it ?

Reply:
tests unit added